### ISSUE TITLE
TypeError: setTimeout(...).unref is not a function

### ISSUE THREAD

**Issue by mifi** [sindresorhus/execa#399]
I use this module all the time including in Electron. After upgrading to the newest version i get this error when trying to call childprocess.kill() from an Electron application:

```
TypeError: setTimeout(...).unref is not a function
setKillTimeout
.../node_modules/execa/lib/kill.js:23:14
```

Note: with execa@1 it works. But 2 or 3 fails with this error. Maybe we could add a check for whether unref exists before calling it, or is Electron not supported anymore?

**Comment by mifi** [comment]
I created a PR to fix

**Comment by ehmicky** [comment]
Thanks a lot @mifi!

**Comment by sindresorhus** [comment]
Fixed by #400

**Comment by ehmicky** [comment]
Published in [`v4.0.0`](https://github.com/sindresorhus/execa/releases/tag/v4.0.0).
